module.yml

```diff
@@ -269,11 +279,63 @@ Resources:
       - 'Fn::ImportValue': !Sub '${AlertingModule}-Arn'
       OKActions:
       - 'Fn::ImportValue': !Sub '${AlertingModule}-Arn'
+  BackupVault: # cannot be deleted with data
```

Comment:
What is the difference between AWS Backup and the property `PointInTimeRecoverySpecification` of the `AWS::DynamoDB::Table` resource? Two separate things? If so, should we also enable `PointInTimeRecoverySpecification`?

Reply:
I assume AWS Backup is creating DynamoDB on-demand backups. So we should enable point-in-time recovery as well?

Reply:
good point. done